docs/spectrum/index.rst

```diff
@@ -29,7 +29,47 @@ Getting Started
 
 TODO
 
-Hello World
+Energy handling in Gammapy
+==========================
+
+.. warning::
+
+   This is  completely experimental
+
+Basics
```

Comment:
Now that we have two classes, it might make more sense to structure this per class instead of per task  (like construction, I/O, computations)?